src/hotspot/os/posix/perfMemory_posix.cpp

```diff
@@ -1028,7 +1028,7 @@ static char* mmap_create_shared(size_t size) {
 //
 static void unmap_shared(char* addr, size_t bytes) {
   int res;
-  if (MemTracker::tracking_level() > NMT_minimal) {
+  if (MemTracker::tracking_level() >= NMT_summary) {
```

Comment:
Just an idle thought, maybe we could add a `MemTracker::is_enabled()` as a shortcut for level > off.

Reply:
I was thinking about doing it in followup, yea, why not here.